dist/accordion/css/accordion.min.css

```
@@ -1 +1,62 @@
-.aria-accordion__panel[hidden]{display:none}
\ No newline at end of file
+/* For legacy browsers support */
+.aria-accordion__panel[hidden] {
+  display: none;
+}
+
+/* For accordion panels with a transition effect */
+.aria-accordion__panel--transition {
+  display: block;
+  max-height: 100vh;
+  visibility: visible;
+  -webkit-transition: max-height 800ms ease-in;
+  -moz-transition: max-height 800ms ease-in;
+  -ms-transition: max-height 800ms ease-in;
+  -o-transition: max-height 800ms ease-in;
+  transition: max-height 800ms ease-in;
+  overflow: visible;
+  /* persist overflow value from animation */
+  animation: 800ms delay-overflow;
+  animation: 900ms delay-padding-expanding;
+}
+
+.aria-accordion__panel--transition[hidden] {
+  display: block;
+  height: auto;
+  max-height: 0;
+  padding-top:0;
+  padding-bottom: 0;
+  overflow: hidden;
+  -webkit-transition: max-height 800ms ease-out;
+  -moz-transition: max-height 800ms ease-out;
+  -ms-transition: max-height 800ms ease-out;
+  -o-transition: max-height 800ms ease-out;
+  transition: max-height 800ms ease-out;
+  visibility: visible;
```

```suggestion
  visibility: hidden;
```
We need `hidden` panel content to avoid screen readers reading it.